### ISSUE TITLE
Support Redragom m601

### ISSUE THREAD

**Issue by Xfezor** [dokutan/mouse_m908#31]
pls support redragon m601 :(

**Comment by TheAquabat** [comment]
I have a 601
```

Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 003: ID 062a:5918 MosArt Semiconductor Corp. 2.4G Keyboard Mouse
Bus 003 Device 002: ID 0c45:5204 Microdia USB DEVICE
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 002: ID 8087:0aa7 Intel Corp. Wireless-AC 3168 Bluetooth
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub

```

**Comment by so1der** [comment]
Hello there! Im also have a m601, more specifically - M601-RGB (Centrophorus 2), but in my case both PID and VID are different. 
```
Bus 001 Device 006: ID 258a:1007 SINOWEALTH Wired Gaming Mouse
```
I tried to add those PID:VID into **include/generic/data.cpp** file and compile it, but I still get a "Couldn't detect mouse" message. It is because I did something wrong, or thats cause mouse doesnt support generic protocol? I also tried a "m601" branch, but ended up with a compilation error, thats why I tried to add PID:VID into default branch.

**Comment by so1der** [comment]
Yup, thats was problem on my side, After correctly recompile it I have a "Couldn't open mouse" message. After thats mouse stop responding, but simple "unplug-plug" brings it back to action. So as far as it looks like, M601-RGB doesnt support this protocol.
